README.md

```diff
@@ -1,2 +1,203 @@
+
+> Describe your project in 1 or 2 sentences.
+
+**[your_project__name]** is a Space Travelers' Hub
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features" />
+
+- **Add**
+- **Subtract**
+- **Multiply**
+- **divsion**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo" />
+
+- [Live Demo Link](https://chic-crepe-c605d4.netlify.app/)
+
+## 💻 Getting Started <a name="getting-started" />
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- React.js
+- Redux/toolkit
+- JavaScript
+- Git
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  cd "to where you want to install your project"
+
+ https://github.com/akhror-valiev/spaceX-group-project
+```
+
+### Install
+
+Install this project with:
+
+```
+npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+npm start
+```
+
+## 👥 Authors <a name="authors" />
+
+👤 **Akhror Valiev**
+
+- GitHub: [GitHub Profile](https://github.com/akhror-valiev)
+- LinkedIn: [LinkedIn Profile](https://www.linkedin.com/in/oshie/)
+
+👤 **Bright Kportiklah**
+
+- GitHub: [GitHub Profile](https://github.com/BryteLitty)
+- LinkedIn: [LinkedIn Profile](https://www.linkedin.com/in/brightkportiklah/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[booking]**
+- [ ] **[Cancel Booking]**
+- [ ] **[connect_to_the_redux_store]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing" />
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support" />
+
+If you like this project support it through staring the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements" />
+
+I would like to thank Microverse for idea of the project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly fix this broken MIT license link to make your readme file more professional.